boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -135,3 +135,6 @@ if(ARCH STREQUAL "i386")
 endif()
 
 include(pcat.cmake)
+if(NOT ARCH STREQUAL "i386" OR NOT (SARCH STREQUAL "pc98" OR SARCH STREQUAL "xbox"))
+    include(uefi.cmake)
+endif()
```

Comment:
Maybe this CMake condition could be done as a single variable like ``if(HAS_UEFI)``, which would be defined elsewhere.